handlers/auth_code_request_update.go

```diff
@@ -160,15 +164,15 @@ func UpdateAuthCodeRequest(authCodeSvc *service.AuthCodeService, authCodeReqSvc
 	})
 }
 
-func sendConfirmationEmail(emailAddress string, r *http.Request, w http.ResponseWriter) {
+func sendConfirmationEmail(emailAddress string, r *http.Request) error {
 	// Send confirmation email
 	if err := handleEmailKafkaMessage(emailAddress); err != nil {
-		log.ErrorR(r, err, log.Data{"email address": emailAddress})
-		w.WriteHeader(http.StatusInternalServerError)
-		return
+		return errors.New("error sending email to " + emailAddress)
```

Comment:
`err` needs to be included in returned error, to make sure that the cause gets logged.